docs/DeveloperGuide.md

```diff
@@ -172,22 +172,23 @@ This section describes some noteworthy details on how certain features are imple
-  /dur 50m /date 21/10/2021 15:10` which calls upon `LogicManager#execute()`
-2. `GameBookParser` and `AddCommandParser` parses the command. If it is valid, a new `GameEntry` object is created,
-  followed by an `AddCommand` object containing the `GameEntry`.
-3. `LogicManager#execute()` calls upon `AddCommand#execute()`. Within `AddCommand#execute()`, `ModelManager#addGameEntry()`
-  is called, which in turn calls `GameBook#addGameEntry()`. This then calls `GameEntryList#add()`, which adds the new game
-  entry to a `List` and sorts it by date.
-4. `AddCommand#execute()` then encapsulates the result of the command execution in a new `CommandResult` object
-  to its caller. The caller, we recall from Step 3, is `LogicManager#execute()`.
-5. To update the storage list, `LogicManager#execute()` then calls `StorageManager#saveGameBook(ReadOnlyGameBook)`,
-  which then calls its overloaded method `StorageManager#saveGameBook(ReadOnlyGameBook, Path)`, which calls
-  `JsonGameBookStorage#saveGameBook(ReadOnlyGameBook, Path)`
-6. Abstracting away the remaining storage details, the new list of game entries is saved in local storage.
-7. The updated list, graph and statistics are reflected in GUI, together with feedback to the user retrieved from
-  the `CommandResult` object from Step 4.
-
+1. The user inputs a command, such as `add /g Poker /p 35 /dur 40m /loc Resort World Sentosa Casino
+   /dur 50m /date 2021-10-21 15:10` which calls upon `MainWindow#executeCommand()`.
+2. `MainWindow#executeCommand()` passes the user's input to `LogicManager#execute()`to process, which calls upon `GameBookParser#parseCommand()`.
+3. `GameBookParser#parseCommand()` parses the input with the help of `AddCommandParser#parse()`. If input is valid, a new `GameEntry` object is created,
+   followed by an `AddCommand` object containing the `GameEntry`. The `AddCommand` object is then returned by `GameBookParser#parseCommand()`. 
+4. `LogicManager#execute()` executes `AddCommand` by calling `AddCommand#execute()`, which adds the new entry to a `List` and sorts it by date.
+5. `AddCommand#execute()` then encapsulates the result of the command execution in a new `CommandResult` object
+   to its caller, `LogicManager#execute()`.
+6. `LogicManager#execute()` calls `Storage` to store the new game entry list and returns `CommandResult` to `MainWindow#executeCommand()`.
+7. `MainWindow#executeCommand()` executes `resultDisplay#setFeedbackToUser()` to display the message from `CommandResult` to the user.
+8. `MainWindow#executeCommand()` calls `StatsPanel#updateStats()` and `GraphPanel#updateGameEntryList()` to update the statistics and graph with the new game entry list.
+
+The following activity and sequence diagrams illustrate the mechanism of adding a new game entry. To reduce clutter, the
+sequence diagram will only focus on Logic and Model components.
+![Activity diagram of an add command](images/AddActivityDiagram.png)
```

Comment:
The images seem a bit small and I can't read the text properly. Not sure if it will be the case also when rendered. Can just swap out the png if is the case.